src/pages/Vote/Governance/index.tsx

```diff
@@ -85,7 +91,8 @@ export const GovernanceUi: React.FC<IGovernanceUiProps> = ({
           ),
         )}
       </div>
-      {total && (
+
+      {total && total > 0 && (
```

Comment:
This is handled by the Pagination component on line 56

Reply:
We need the additional check because of the placeholder data I added. I added them to be consistent with how we use pagination on other pages